lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaThirdPartyDocumentGateway.java

```diff
@@ -117,6 +117,12 @@ public DocumentCheckResult performDocumentCheck(DrivingPermitForm drivingPermitD
         // Here (for the DVA request) it needs to be mapped back to date of issue
         dvaPayload.setDateOfIssue(drivingPermitValidFrom);
 
+        try {
```

Comment:
This should be removed or behind an appropriate parameter

Reply:
Ah whoops that log line shouldn't even be there